build/root/BUILD.root

```diff
@@ -35,11 +36,11 @@ gcs_upload(
         "//cluster/gce/gci:gcs-release-artifacts-and-hashes",
     ],
     tags = ["manual"],
-    upload_paths = {
-        "//:_binary-artifacts-and-hashes": "bin/linux/amd64",
+    upload_paths = select(for_platforms(for_all = {
```

Comment:
AKA I haven't used select much before, so would be useful to provide some extra documentation as essentially a tutorial for people who will be supporting this.

Reply:
done